cmd/pbm/status.go

```diff
@@ -543,7 +545,7 @@ func getStorageStat(
 	s.Region = cfg.Storage.Region()
 	s.Path = cfg.Storage.Path()
 
-	bcps, err := pbm.GetAllBackups(ctx)
+	bcps, err := backup.BackupsList(ctx, conn, profile, 0)
```

Comment:
This filtered backup list doesn't contain base backups for PITR, and then PBM shows something like:
```
pbm s --profile s3

 Snapshots:
    2025-12-05T13:29:01Z 270.03KB <logical, s3> success [restore_to_time: 2025-12-05T13:29:05]   
  PITR chunks [7.51MB]:
    2025-12-05T13:25:32 - 2025-12-05T16:00:52
    2025-12-05T13:24:29 - 2025-12-05T13:25:31 (no base snapshot) <--
```
The best should be to omit the list of PITR in case of --profile, so the same as we do for `list`. Or pass down the full list of backups, so that it's possible to make that resolution.